contracts/contracts/RaidenMicroTransferChannels.sol

```diff
@@ -223,9 +235,8 @@ contract RaidenMicroTransferChannels {
         uint32 _open_block_number,
         uint192 _added_deposit)
         external
+        isTrustedContract
```

Comment:
this is a style thing and I don't think we got it specified anywhere but I believe it's easier to read if the visibility specifier comes last and before it we got the modifiers.

Reply:
This was the style I used for the auction contract: visibility, then modifiers.
But it makes sense to make the visibility specifier more.. visible.